dlgr/griduniverse/experiment.py

```diff
@@ -742,24 +751,28 @@ def consume(self):
         consumed = 0
         for player in self.players.values():
             position = tuple(player.position)
-            if position in self.food_locations:
-                food = self.food_locations[position]
+            if position in self.object_locations:
+                food = self.object_locations[position]
+                # any object with calories is food for now. Other objects are ignored.
```

Comment:
I don't think this is quite right. "Food" is any non-interactive object with calories, so:
```
item_info = self.object_config.get(obj_id)
is_food = item_info.get('calories', 0) and not item_info.get('interactive', True)
```

Maybe this can be a dynamic property of `Object`.